membership-attribute-service/app/services/AttributesMaker.scala

```diff
@@ -55,7 +56,9 @@ class AttributesMaker {
     }
 
     maybeAlertAvailable.getOrElse(Future.successful(None)) map { maybeAlert =>
-      maybeAlert map { alert => SafeLogger.info(s"User $identityId has an alert available: $alert") }
+      def customFields(identityId: String, alertAvailableFor: String) = List(LogFieldString("identity_id", identityId), LogFieldString("alert_available_for", alertAvailableFor))
+
+      maybeAlert map { alert => logInfoWithCustomFields(s"User $identityId has an alert available: $alert", customFields(identityId, alert)) }
```

Comment:
you might get a warning from this - you can use foreach instead of map to avoid that